google/cloud/bigtable/client.py

```diff
@@ -54,6 +54,11 @@
 from google.cloud.bigtable.cluster import _CLUSTER_NAME_RE
```

Comment:
In https://github.com/googleapis/python-bigtable/issues/1157, I also suggested adding a `client.table()` reference as part of the [Table class](https://github.com/googleapis/python-bigtable/blob/044efe74e0ae8eae46d5567276b5a5d9b699c315/google/cloud/bigtable/table.py#L129). You should probably do that in this PR as well